### ISSUE TITLE
Message passing example does not work reliable with long commands

### ISSUE THREAD

**Issue by blahgeek** [akermu/emacs-libvterm#283]
I followed the instructions about message passing in README and defined a `find-file` function in shell, it generally works, but does not work reliable with long file names.

It seems that the `vterm_cmd` example from README prints out the messages in separated commands, which may generates multiple callbacks in vterm and breaks parsing.

I changed the function to the followings and it works better:

```
OUTSTR="$(vterm_cmd find-file "$(realpath "$@")")"
echo "$OUTSTR"
```

However, It's still not guaranteed that single echo command will generate single callback in vterm. The parsing algorithm inside vterm need to support long message passing consists of multiple callbacks.

**Comment by Sbozzolo** [comment]
Thanks, see comment in #314. A fix is coming but it will require your intervention

**Comment by Sbozzolo** [comment]
Can you please check the new suggested code in #335?

**Comment by Sbozzolo** [comment]
Closed with #335
